gf-guide/metrics.md

```diff
@@ -31,11 +31,21 @@ Please note that the first metrics guidelines referred to in this guide are Lati
 
 The following rules apply to all new font families, and should be enforced to upgraded font families when possible.
 
-**1. Vertical metrics must not be calculated by the font editor automatically.**
+#### 1. Vertical metrics must not be calculated by the font editor automatically
 
-GF doesn't want this because they all do it differently.
+This is an overall best practice in design. When defining line spacing, it's important to strike a balance between legibility and avoiding overlapping letters. At GF, we have established a specific system that prioritizes optimal performance for various situations, while still leaving room for future update improvements like adding more languages without causing regressions. 
 
-**2. The following vertical metric parameters must be set for each font in a family.**
+#### 2. Vertical metrics must be consistent across a family.
+
+Each font in a family must share the same vertical metric values.
```

Comment:
Vertical metrics should be plural to avoid confusion.

Reply:
Done